pkgs/build-support/setup-hooks/install-shell-files.sh

```diff
@@ -219,6 +219,11 @@ installShellCompletion() {
         else
             install -Dm644 -T "$arg" "$outPath"
         fi || return
+
+        if [ $(stat -c %s "$outPath") -eq 0 ]; then
```

Comment:
if this is checks the size, bash has a builtins for that:
```
       -s file
              True if file exists and has a size greater than zero.
```

Reply:
Thanks for the tip, will use that instead!